ios/ParentBuddyAppXk-Info.plist

```diff
@@ -46,6 +46,16 @@
 				<string>bebbo</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>Bundle ID</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>org.unicef.kosovo.foleja</string>
```

Comment:
Is this change going to the Foleja build?

Reply:
yes this will be in foleja build as for bebbo prod build we have already added.